iroh/src/baomap/flat.rs

```diff
@@ -865,7 +875,13 @@ impl Store {
             tracing::info!("copying {} to {}", source.display(), target.display());
             progress(0)?;
             // todo: progress
-            std::fs::copy(&source, &target)?;
+            let size = if let Some(size) = reflink::reflink_or_copy(&source, &target)? {
+                tracing::trace!("copied {} to {}", source.display(), target.display());
+                size
+            } else {
+                tracing::trace!("reflinked {} to {}", source.display(), target.display());
+                std::fs::metadata(&target)?.len()
```

Comment:
we do this just to get the size to report progress. Maybe not do it?

Reply:
It's not particularly expensive, and progress bars are a great thing. So anything that is really gained from not doing it?